### ISSUE TITLE
preview of theme is not working correctly

### ISSUE THREAD

**Issue by rallek** [zikula/core#3957]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 2.012
| PHP Version       | 7.1

### Expected behavior
If I am using the preview of a theme it should look the same when I make it the default theme

### Actual behavior
the theme is using all but not the css. If I have as default my standard bootstrap theme and my default theme is in another color and do have e.g. a logo inside the menue the preview is showing up with the new logo inside the menü but with the color set of the current active theme.

### Steps to reproduce
see behavior

**Comment by Guite** [comment]
Even if another theme declares its own `bootstrapPath` in `theme.yml`, the calling/default theme's stylesheet is additionally included in the preview (here: `<link rel="stylesheet" href="/themes/BootstrapTheme/Resources/public/css/cerulean.min.css" type="text/css">`).

**Comment by craigh** [comment]
this entire feature is currently removed.

**Comment by craigh** [comment]
also bb30a9568d237399b8a50c81b336a219872da234

**Comment by craigh** [comment]
I have no idea if this ticket is still valid. part of the reason is I have no non-core theme to check it with

**Comment by Guite** [comment]
> if this ticket is still valid

it very probably is

**Comment by Guite** [comment]
#3999 is a duplicate and was reported two months ago
also basically nothing changed since 2.x regarding to how the preview works

**Comment by craigh** [comment]
I believe this ticket is now fixed. I've built and tested theme to confirm. The main theme is ZikulaBootstrapTheme (See first screenshot) and the 'previewed' theme is [SimpleScrollTheme](https://github.com/craigh/SimpleScrollTheme). The theme is built using the standard bootstrap file (not customized). So it doesn't use the custom bootstrap parameter setting in `theme.yaml`. So in this case, there is no appearance of the currently active (cerulean) version of the bootstrap file. (See second screenshot). Then I added a custom bootstrap build to the theme and set the param in `theme.yaml`. In this case there is also no appearance of the currently active (cerulean) version of the bootstrap file. Conclusion: fixed!

![ss1](https://user-images.githubusercontent.com/350048/78936669-07e26680-7a7d-11ea-87bb-40719d546517.png)
![ss2](https://user-images.githubusercontent.com/350048/78936679-0add5700-7a7d-11ea-8997-c0e94e181a8f.png)
![ss3](https://user-images.githubusercontent.com/350048/78936682-0ca71a80-7a7d-11ea-91f1-f74a5c239877.png)
